README.md

```diff
@@ -114,6 +114,43 @@ More examples can be found here:
 
 <center><a href="https://sfu-db.github.io/dataprep/connector.html#getting-web-data-with-connector-query"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector.png"/></a></center>
 
+### Clean
+
+DataPrep.Clean contains simple functions designed for cleaning and standardizing a column in a DataFrame. It provides
+* A unified API: each function follows the syntax `clean_{type}(df, "column name")` (see an example below)
+* PyData support: its design for cleaning pandas and Dask DataFrames enables seamless integration into the Python data science workflow
```

Comment:
Do other projects use `PyData` to refer to the python data science workflow?

Reply:
I think [sparse](https://github.com/pydata/sparse) does: "multi-dimensional arrays for the PyData ecosystem". But I wrote PyData here since I thought it was consistent with our website:
```
Integrate Seamlessly with PyData
——————————————————————
DataPrep is built on Pandas/Dask DataFrame thus can be seamlessly integrated with other Python libraries.
```
But it can also refer to the PyData conference. Do you have any other ideas or suggestions here?

Reply:
My suggestion is that we just say something simple and explicitly like "Python DataFrame Support" or "Python Data Science Support" here to avoid confusion

Reply:
Yea I agree with saying something more explicit. PyData is confusing. Someone may don't know what is pydata.